tests/repro_tests/frontend/verifier/test_verifier_reproducibility_base.py

```diff
@@ -0,0 +1,70 @@
+""""Reproducibility tests for verifiers.
+
+Authors: Ayush Baid
+"""
+import unittest
+from pathlib import Path
+from typing import Tuple
+
+from gtsam import Rot3, Unit3
+
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.geometry_comparisons as geometry_comparisons
+from gtsfm.frontend.detector_descriptor.sift import SIFTDetectorDescriptor
+from gtsfm.frontend.verifier.ransac import Ransac
+from gtsfm.frontend.verifier.verifier_base import VerifierBase
+from gtsfm.frontend.matcher.twoway_matcher import TwoWayMatcher
+from tests.repro_tests.test_repro_base import ReproducibilityTestBase
+
+# defining the path for test data
+TEST_DATA_ROOT_PATH = Path(__file__).resolve().parent.parent.parent.parent / "data"
+IMG1_PATH = TEST_DATA_ROOT_PATH / "crane_mast_8imgs_colmap_output" / "images" / "crane_mast_1.jpg"
```

Comment:
@ayushbaid have we checked these in?

Reply:
I did not. Let me do it now.

Reply:
Thanks for adding these. Do we need to check in all of them @ayushbaid? Each is 5 MB, so we would be adding 40 MB to the repo, which I think may slow down push/pull more than we want.

How about just 2 of them?

Reply:
we might want to purge the other 6 from the git history, since otherwise they'll be embedded in the commits

Reply:
Makes sense. Let me do this.

Reply:
Or I can just squash and merge this PR?

Reply:
that works, thanks